src/stomp_moveit_planning_context.cpp

```diff
@@ -13,16 +13,24 @@
 #include <stomp_moveit_parameters.hpp>
 
 #include <moveit/constraint_samplers/constraint_sampler_manager.h>
+#include <moveit/robot_state/conversions.h>
 
 namespace stomp_moveit
 {
 bool solveWithStomp(const std::shared_ptr<stomp::Stomp>& stomp, const moveit::core::RobotState& start_state,
```

Comment:
For readability, I would suggest creating a separate function "adaptWithStomp()" that takes the input trajectory instead of start and goal states.